doc/Food.rst

```diff
@@ -77,3 +81,5 @@ If there are a lot of crumbs and things lying around, please wipe down the surfa
 Reimbursement
 -------------
 Email Dorit with a PDF of your receipt as well as a list of each person who ate pizza and their titles (ex: "Graduate Student", "Professor", "Postdoc", etc). CC Melissa or whoever paid. In the body of your email, mention the total cost and explain who should be reimbursed.
+
+It's best to remove any personal information from PDF receipts before you submit them for reimbursement. Before printing your email receipt as a PDF, you should feel free to delete any information like your phone number or email address. You can use the `Inspect Element <https://devtoolstips.org/tips/en/remove-annoying-overlays>`_ trick to do this.
```

Comment:
Can I suggest this link https://yamm.com/blog/how-to-print-email-from-gmail-without-header/ instead? I think it better matches what you explained to me.

Reply:
ooh yeah, that one is much better! good find